Website rework magenta core

norahueppi opened the pull request:
the goal was to update the current website with the informations of the matrix project to the new magentacore project informations. I changed:
- I adjusted all the category
- changed the pictures
- adjusted the texts
- made a new category for all the old project informations
-  updated the pdf files
- and the software installtion manual
- made one example with the magenta library  (the others dont work, have to update them)

norahueppi committed:
add new category and move old infomations in that category

norahueppi committed:
update parts list

norahueppi committed:
add links to change directories

norahueppi committed:
update the paths for the old pictures

norahueppi committed:
change infos to new project

norahueppi committed:
change linkes from title3 to bold

norahueppi committed:
update pictures for pcb

norahueppi committed:
Update index.html

norahueppi committed:
update parts list

norahueppi committed:
add pdf of new project and made folders for the matrix pdf and a folder  for the magenta pdf

norahueppi committed:
change the contact email

norahueppi committed:
update the Stückliste text

norahueppi committed:
update Schema & Bestückungsplan

norahueppi committed:
change name exapmles and added a magentacore version of bar-counter.c

norahueppi committed:
add pictures of the new case

printed case and 3D model

norahueppi committed:
add magentacore manual

norahueppi committed:
add  pictures for vor installation for vscode and magentacore library

norahueppi committed:
add three new categories for the software installation

norahueppi committed:
update software documentation

norahueppi committed:
change title from Matrix project to Magenta Core project

norahueppi committed:
change software version

norahueppi committed:
change github links

norahueppi committed:
change the name if the link to the index an delete the links to the bootloaders and ftdi

norahueppi committed:
delete some line break

norahueppi committed:
chenge index to magenta core informations

norahueppi committed:
Change resources

norahueppi committed:
change software-installation

norahueppi committed:
add category software-template-installation

norahueppi committed:
add category software-new-project-installation

norahueppi committed:
add category software-code-compile-guide

norahueppi committed:
Update software-introduction to current project

norahueppi committed:
update software-examples to current project

norahueppi committed:
change weiteres-expansion to magentacore project

norahueppi committed:
update assembly-placement to magentacore project

norahueppi committed:
update assembly-housing to magentacore

norahueppi committed:
delete assembly-botloadersfuses and assembly-ftdi

norahueppi committed:
change assembly -test to current project

norahueppi committed:
change spelling mistake

norahueppi committed:
change spelling mistake

norahueppi committed:
fix spelling mistake

norahueppi committed:
delete unnecessary routes and add new one for bom

norahueppi committed:
change shematic pdf

they have now better quality

norahueppi committed:
change layout pdf

has now better quality

norahueppi committed:
add ibom.html

now schnuppis can solder the print easier at home

norahueppi committed:
check spelling

norahueppi committed:
add category for the ibom

norahueppi committed:
changed link to library releases

norahueppi committed:
describe the functions better

norahueppi committed:
rewrote the sentence fpr Pos and what the schnuppis should do with the "Schon Bestücht" partlist

norahueppi committed:
fix miss spelling

norahueppi committed:
add structure to add the repository of the test software

norahueppi committed:
change formatting

norahueppi commented:
<h2> Übersicht -> Hilfreiche Unterlagen</h2> 

- checked spelling in the pdf file
- the Magenta Core Library Download is now pionted to the releases


<h2> Hardware -> Schema & Bestückungsplan</h2>

- corrected the sentence which descibed "Pos."  
- mentioned that there are pre-assembled parts, that the "Schnuppis" dont have to solder

<h2> Hardware -> Schema & Bestückungsplan</h2>

- changed pdf files for layout and schematic, they were made vio print option in KiCad not via printscreen
- added category for the BoM

<h2>Software -> Einführung</h2>

- When a function is mentioned there is always "()"

<h2>Software -> Dokumentation</h2>

- explained what API is
- put a title to every function and variable i describe
- described the functions better
- rewrote the senstence on the bottom "Aufgerufen werden können diese beispielsweise via magentaobj.init(); für Methoden oder z.B. magentaobj.button_1 um einen Tastenwert zu erhalten."


<h2>Zusammenbau -> Test</h2>

- added the structure to add the link to the software test repository (have to make a repository)

<h2> Matrix (1.x)</h2>
<B>Software</B>

- changed the magentaobj...something back to matrix...something

norahueppi committed:
change library name to Matrix_RKAG.h

norahueppi committed:
update and rename BoM category

- renamed the category
- updated the BoM so that only the componends are shown that the Schnuppi have to solder

norahueppi committed:
change pdf to better quality

norahueppi committed:
rewrote sentences

norahueppi committed:
explain some variables and functions better

norahueppi committed:
rewrote download instruction

norahueppi committed:
correct spelling mistake

norahueppi committed:
correct spelling in Unterlagen.pdf

corrected the word Platform DIE to Platform IDE

mburger82 commented:
## Unterlagen.pdf:
- Check last sentence on page 53

## Software - Installation:
<img width="256" height="45" alt="image" src="https://github.com/user-attachments/assets/c5346d68-734f-4689-9c71-a5ddf2ef8c43" />

## Software - Beispiele
<img width="717" height="72" alt="image" src="https://github.com/user-attachments/assets/6984b0f0-10ca-4ae3-81f6-fdda302f9d5c" />

<img width="1017" height="52" alt="image" src="https://github.com/user-attachments/assets/b2108791-eac2-40d9-81dc-c28f10ae47ee" />

<img width="452" height="58" alt="image" src="https://github.com/user-attachments/assets/058caec6-f62d-49eb-a7cd-afebb0a1247f" />

<img width="577" height="40" alt="image" src="https://github.com/user-attachments/assets/66dcad14-e648-4cde-9866-92e57c730ed9" />

<img width="1016" height="49" alt="image" src="https://github.com/user-attachments/assets/a0ba4f41-6c09-4125-a426-7cc36a936168" />
- The statement is furthermore not completely correct. You cannot use the matrix library with the new magenta core hardware, but you may use the objectname "matrix" within a magenta core project because both objectnames "magentaobj" and "matrix" are available.

norahueppi committed:
rewrote the last sentence on page 53

norahueppi committed:
change the IDE in the installation description

norahueppi committed:
update software- Einführung

- changed the title
- corrected spelling mistakes
- rewrote semnetnce which explained loop function

norahueppi committed:
correct spelling mistake

norahueppi committed:
correct spelling mistake
